perun-core/src/main/java/cz/metacentrum/perun/core/entry/GroupsManagerEntry.java

```diff
@@ -52,7 +52,7 @@ public Group createGroup(PerunSession sess, Vo vo, Group group) throws GroupExis
 
 		// Authorization
 		if (!AuthzResolver.isAuthorized(sess, Role.VOADMIN, vo) &&
-		    !AuthzResolver.isAuthorized(sess, Role.TOPGROUPCREATOR, vo)) {
+			!AuthzResolver.isAuthorized(sess, Role.TOPGROUPCREATOR, vo)) {
```

Comment:
This fixing of code indentation is not need for this pull request. Please remove it.